### PR TITLE
Add documentation to the repo

### LINKED ISSUE #25: MachineLearning_Toolkit: Installation process
#### What is missing/incorrect?

A documentation of the installation process is needed to get user started, before we have options in installer. Maybe on wiki page. 
1. InstallPythonToolkit component
2. InstallMachineLearning component
3. Close\Reopen Rhino and Grasshopper

This may be needed for Python_Toolkit as well.

### LINKED ISSUE #30: Document the procedure to add a new ML object/algorithm

### PR DESCRIPTION
**Pull request by epignatelli**
### Issues addressed by this PR

Closes #25
Closes #30

### Changelog

- Add documentation about installation procedure
- Add documentation on how to add another algorithm

### CODE REVIEW

**Review by BingWangUS — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,45 @@
...
+1. Open a UI of your choice (e.g. Grasshopper)
+1. Run the `BH.Engine.Python.Compute.InstallPythonToolkit` component and wait for the installation to finish.
+1. The installation has succeeded if the install packages include:
+	  - Python 3.7
+  	- jupyter
+  	- matplotlib
+  	- Python_Toolkit
+1. Restart your UI - if using Grasshopper, restart Rhinoceros and Grasshopper
+1. Reopen the UI and now install the python bindings of the MachineLearning_Toolkit by running
+   the `BH.Engine.Python.Compute.InstallMachineLearningToolkit` component
```
Comment (BingWangUS) [inline README.md]: `BH.Engine.MachineLearning.Compute.InstallMachineLearningToolkit`

File: README.md [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
+1. Restart your UI
+   
+Note that the intallation will take a while - around 15 minutes.
```
Comment (BingWangUS) [inline README.md]: typo - installation

**Review by BingWangUS — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (epignatelli) [reply README.md]:
```suggestion
   the `BH.Engine.MachineLearning.Compute.InstallMachineLearningToolkit` component
```

File: README.md (hunk 2 above)
Comment (epignatelli) [reply README.md]:
```suggestion
Note that the installation will take a while - around 15 minutes.
```

**Review by BingWangUS — APPROVED** [review]

**Pull request merged**